fahrschein/build.gradle

```diff
@@ -12,6 +12,11 @@ dependencies {
     implementation "com.fasterxml.jackson.core:jackson-annotations:${property('jackson.version')}"
     implementation "com.fasterxml.jackson.datatype:jackson-datatype-jdk8:${property('jackson.version')}"
     implementation "com.fasterxml.jackson.datatype:jackson-datatype-jsr310:${property('jackson.version')}"
+    compileOnly 'org.projectlombok:lombok:1.18.26'
```

Comment:
please don't add lombok for this

Reply:
General idea is to ease development efforts as I can see there were lots of other classes which could have been greatly simplified with Lombok usage. But I can move this discussion as part of another issue. 👍

Reply:
Removed lombok